lib/simulation/openfisca/mapping/famille.js

```diff
@@ -10,9 +11,10 @@ module.exports = {
     },
     enfants: {
         fn: function(situation) {
-            var enfants = _.pluck(_.filter(situation.individus, {role: 'enfant'}), '_id');
-            var personnesACharge = _.pluck(_.filter(situation.individus, {role: 'personneACharge'}), '_id');
-            return enfants.concat(personnesACharge); 
+            var enfants = _.filter(situation.individus, function(individu) {
+                return mapping.isIndividuValid(individu, situation) && individu.role == 'enfant';
+            });
+            return _.pluck(enfants, '_id');
```

Comment:
This code is duplicated all over the place. Now that you've added a dependency to `index.js` for `isIndividuValid`, go all the way and factor in that function  ;)